templates/CONFIG_TEMPLATE.INI

```diff
@@ -360,13 +357,7 @@
 
 #  sets the space token for storing temporary files (e.g. oversized input
 #  sandbox)
-#DefaultSRMToken = 
-
-#  enables/disables the support of the EDG middleware
-#EDG_ENABLE = True
-
-#  sets the LCG-UI environment setup script for the EDG middleware
-#EDG_SETUP = /afs/cern.ch/sw/ganga/install/config/grid_env_auto.sh
+#DefaultSRMToken =
```

Comment:
Minor point not about the changes, actually more like a comment...I'm pretty sure that we don't use these templates any more as the gangarc is generated from the config object itself. Do we want to be keeping them up to date or should we have a separate issue for removing them.